tutorials/notebooks/example_keras_effdet_lite0.ipynb

```diff
@@ -107,6 +107,7 @@
    "source": [
     "import sys\n",
     "sys.path.insert(0,\"/content/local_mct\")\n",
+    "!pip install -r /content/local_mct/requirements.txt\n",
```

Comment:
maybe also run `pip uninstall mctq` and then get the latest code of mctq quantizers?

Reply:
we assume mctq version is updated in "requirements.txt"